src/Microsoft.ML.HalLearners/Microsoft.ML.HalLearners.csproj

```diff
@@ -1,8 +1,9 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <IncludeInPackage>Microsoft.ML.HalLearners</IncludeInPackage>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
> <AllowUnsafeBlocks>true</AllowUnsafeBlocks> [](start = 3, length = 44)

think i removed this

Reply:
I need it for symsgd.

---
In reply to: [206926582](https://github.com/dotnet/machinelearning/pull/624#discussion_r206926582) [](ancestors = 206926582)